AI-Social-Platform.Server/appsettings.json

```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=DESKTOP-GIE8HH7\\SQLEXPRESS;Database=AI-Social-Platform;Trusted_Connection=True;MultipleActiveResultSets=True"
```

Comment:
Please add comment in file in possible - that the connection string(server) should be changed by each teammate when they get the changes. I don't remember if json file allowed comments - if not - please write it in the communication channel and tag everyone.

Reply:
Sorry, my mistake. I fixed it.